common/djangoapps/philu_commands/management/commands/send_ondemand_reminder_emails.py

```diff
@@ -0,0 +1,191 @@
+
+                # Check if user has submitted last modules graded oras or not. If yes not need to send email OR
+                # If user's submission gets equal to graded oras count than don't need to continue..
+                if (last_module_oras and check_for_last_module_submission(last_module_oras, anonymous_user)) or \
+                        len(response_submissions) == graded_oras_count:
+                    continue
+
+                latest_submission = response_submissions.first()
+                # Count to keep track of how many times user shows an inactivity for "INACTIVITY_REMINDER_DAYS"
+                time_passed_count = 0
+
+                if len(response_submissions) == 0 and has_time_passed_since_activity(enrollment.created.date(), today):
+                    send_reminder_email(user, course, course_deadline)
+                    continue
+                elif len(response_submissions) > 0:
+                    # Check for latest submission entry from submission table if the difference of created date and
+                    # today is equals to "INACTIVITY_REMINDER_DAYS" days this means that email should be send to user
+                    # but before that we need to check is it the first time user shows an inactivity for
+                    # "INACTIVITY_REMINDER_DAYS" days or not if yes than send email else means that emails has already
+                    #  been sent to user so no need to send it again.
+                    if has_time_passed_since_activity(latest_submission.created_at.date(), today):
+                        last_response_time = latest_submission.created_at.date()
+                        # We have checked first entry separately so starting from second index.
+                        for index_response, response in enumerate(response_submissions[1:]):
+                            if has_time_passed_since_activity(response.created_at.date(), last_response_time):
+                                time_passed_count += 1
+                        # If user haven't been inactive in past before today than send email.
+                        if time_passed_count == 0:
+                            send_reminder_email(user, course, course_deadline)
+                    else:
+                        continue
+
+
+def has_time_passed_since_activity(first_date, second_date):
+    return True if (second_date - first_date).days == INACTIVITY_REMINDER_DAYS else False
+
+
+def get_suggested_course_deadline(enrollment_date, chapters):
+    return enrollment_date + timedelta(days=(len(chapters) * DAYS_FOR_EACH_MODULE))
+
+
+def get_graded_ora_count(oras_block):
+    graded_oras = []
+    for ora in oras_block:
+        if ora['graded']:
+            graded_oras.append(ora)
+    return len(graded_oras)
+
+
+def get_all_oras(course_struct):
+    ora_blocks = []
+    for k, v in course_struct['blocks'].iteritems():
```

Comment:
lets add spaces before for and return statements, in the whole file

Reply:
lets change this from `for k, v in course_struct['blocks'].iteritems()` to `for block in course_struct['blocks'].itervalues()